src/storage/admin/AdminTaskManager.cpp

```diff
@@ -111,10 +111,11 @@ void AdminTaskManager::schedule() {
         auto task = it->second;
         auto errOrSubTasks = task->genSubTasks();
         if (!nebula::ok(errOrSubTasks)) {
-            LOG(ERROR) << "genSubTasks() failed=" << static_cast<int>(nebula::error(errOrSubTasks));
+            auto code = static_cast<int>(nebula::error(errOrSubTasks));
+            LOG(ERROR) << "genSubTasks() failed, err=" << code;
```

Comment:
A little suggestion,
Should I print the task name of the task into the log?

Reply:
good catch, fixed